app/assets/stylesheets/components/modals.css.scss

```diff
@@ -81,6 +81,8 @@ Full license explanation at https://github.com/houdiniproject/houdini/blob/main/
 	top: 10px;
 	right: 10px;
 	width: 24px;
+	background-color: transparent;
+	padding: 0;
```

Comment:
What was the reasoning for this change?

Reply:
With the change in the file "_modal_close.html", the layout of the button was affected. And because of that, I decided to update the CSS.